mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+		METHOD m_ilcdavkn createUnseeded (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_mdkmcezi$C_uejbabyt$C_zlreguhf;DD)Lnet/minecraft/unmapped/C_mdkmcezi$C_uejbabyt;
+			ARG 0 continentalness
+			ARG 1 erosion
+			ARG 2 weirdness
+			ARG 3 spline
+			ARG 4 minValue
+			ARG 6 maxValue
+		METHOD m_ncjdwzpl weirdness ()Lnet/minecraft/unmapped/C_kzwapyud;
+		METHOD m_rhusmglq spline ()Lnet/minecraft/unmapped/C_mdkmcezi$C_uejbabyt$C_zlreguhf;
+		METHOD m_sxtwqolz shaper ()Lnet/minecraft/unmapped/C_vtphaipm;
+		METHOD m_uwowgrpw continentalness ()Lnet/minecraft/unmapped/C_kzwapyud;
+		CLASS C_gooqqxmh
+			METHOD apply apply (Lnet/minecraft/unmapped/C_vtphaipm;Lnet/minecraft/unmapped/C_vtphaipm$C_wfilltqw;)F
+		CLASS C_zlreguhf Type
+			FIELD f_ovnzrvtd CODEC Lcom/mojang/serialization/Codec;
+			FIELD f_uecpyigc type Ljava/lang/String;
+			METHOD <init> (Ljava/lang/String;ILjava/lang/String;Lnet/minecraft/unmapped/C_mdkmcezi$C_uejbabyt$C_gooqqxmh;)V
+				ARG 3 type
+	CLASS C_ufxlyrzf WeirdScaledSampler
+		FIELD f_bgfgqhdk input Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_lcvlminl noiseData Lnet/minecraft/unmapped/C_cjzoxshv;
+		FIELD f_spincsyf CODEC Lcom/mojang/serialization/Codec;
+		FIELD f_tfqmetif noise Lnet/minecraft/unmapped/C_rxrtfjrt;
+		FIELD f_vkgxqwyx rarityMapper Lnet/minecraft/unmapped/C_mdkmcezi$C_ufxlyrzf$C_edgplzwm;
+		FIELD f_zgnjzonn DATA_CODEC Lcom/mojang/serialization/MapCodec;
+		METHOD <init> (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_rxrtfjrt;Lnet/minecraft/unmapped/C_mdkmcezi$C_ufxlyrzf$C_edgplzwm;)V
+			ARG 1 input
+			ARG 2 noiseData
+			ARG 3 noise
+			ARG 4 rarityMapper
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_astgbggx createUnseeded (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_mdkmcezi$C_ufxlyrzf$C_edgplzwm;)Lnet/minecraft/unmapped/C_mdkmcezi$C_ufxlyrzf;
+			ARG 0 input
+			ARG 1 noiseData
+			ARG 2 rarityMapper
+		METHOD m_lheavxto noise ()Lnet/minecraft/unmapped/C_rxrtfjrt;
+		METHOD m_nhparupd rarityMapper ()Lnet/minecraft/unmapped/C_mdkmcezi$C_ufxlyrzf$C_edgplzwm;
+		METHOD m_xmgcvzhq noiseData ()Lnet/minecraft/unmapped/C_cjzoxshv;
+		CLASS C_edgplzwm Type
+			FIELD f_bknlxpxb type Ljava/lang/String;
+			FIELD f_pslonplh CODEC Lcom/mojang/serialization/Codec;
+			METHOD <init> (Ljava/lang/String;ILjava/lang/String;Lit/unimi/dsi/fastutil/doubles/Double2DoubleFunction;D)V
+				ARG 3 type
+	CLASS C_vbvgjlqq Clamp
+		FIELD f_bcukjvej minValue D
+		FIELD f_lmwrlosq DATA_CODEC Lcom/mojang/serialization/MapCodec;
+		FIELD f_ofoibhfv input Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_relqvmbd CODEC Lcom/mojang/serialization/Codec;
+		FIELD f_ziljolra maxValue D
+		METHOD <init> (Lnet/minecraft/unmapped/C_kzwapyud;DD)V
+			ARG 1 input
+			ARG 2 minValue
+			ARG 4 maxValue
```

Comment:
I feel as if I'm just missing something, but this arg number doesn't seem right either

Reply:
Concern, it might be normal idk